README.md

```diff
@@ -20,25 +20,43 @@ The following commands are supported for both Maven and Gradle Quarkus projects:
   * `Quarkus: Add extensions to current project`: Add Quarkus extensions to currently opened Quarkus project
   * `Quarkus: Debug current Quarkus project`: Launches the Maven `quarkus:dev` plugin or the Gradle `quarkusDev` command and automatically attaches a debugger  
 
-## Quarkus `application.properties` Features
-  * Completion support for Quarkus properties
-  * Hover support for Quarkus properties
-  * Validation support for Quarkus properties 
-  * Support for Quarkus profiles
-  * Outline support (flat or tree view)
+## Quarkus / MicroProfile `properties` Features
```

Comment:
I think it is better to remove spaces before and after forward slashes:
https://www.grammarly.com/blog/slash/
https://english.stackexchange.com/questions/2131/should-i-write-module-theme-or-module-theme

Reply:
ie, Quarkus/MicroProfile instead of Quarkus / MicroProfile

Reply:
fixed

Reply:
Thanks @xorye for the links